app/controllers/application_controller.rb

```diff
@@ -21,4 +23,10 @@ def current_tenant(current_user)
     return tenant if tenant
     raise  Exceptions::NoTenantError
   end
+
+  def check_entitled(entitlement)
+    required_entitlements = %i[hybrid_cloud?]
```

Comment:
This makes the most sense for now.  At some point in the future we'll need to open this up when approval supports other services, but there is no current timeframe for that.